components/molecule/checkboxField/src/index.js

```diff
@@ -15,13 +15,15 @@ const MoleculeCheckboxField = ({
   alertText,
   helpText,
   onChange,
+  isBoldLabel,
```

Comment:
if this prop is not required it would be nice to have a default prop like `false`